recipes/hunspell/post-link.sh

```diff
@@ -0,0 +1,7 @@
+# Unfortunately, this can't detect if you actually did install a dictionary
+# alongside aspell, as hunpell would be a dependency of such a package and
+# would be linked first.
+if [ ! -e "$PREFIX/conda-meta/aspell-[!0123456789]*" ]; then
```

Comment:
Cna you explain why this is needed?

Reply:
It is in the comment above, but basically, hunspell doesn't come with any dictionaries.  Those have to be separately packaged.  This is a reminder to users to install a dictionary if they don't have any installed already.

Reply:
What about creating those conda packages first and adding them as a dependency for this package?

Reply:
I think that is a bit backwards.  As an analogy, we wouldn't have an NLTK data package that is a dependency of  NLTK.  NLTK can function just fine without the stock data they provide.  Similarly here, not everyone needs every dictionary, or any particular dictionary (e.g. English). If anything, the data packages depend on the software that uses them.

Reply:
Got it

Reply:
Should it be `aspell` or `hunspell` in the above line?

Reply:
I think that this is correct because aspell and hunspell can share dictionaries

Reply:
I saw in @asmeurer's PR that there can be `hunspell-en. Then a `or` clause checking for `aspell-[!0123456789]*` and `hunspell-[!0123456789]*` would be better.

Reply:
Done

Reply:
The separate dictionary packages mirrors what other distributions do. It makes sense if you consider that they may potentially be many language dictionaries, but you don't want to install all of them. Perhaps it's a stupid optimization since the dictionaries aren't really that big (my hunspell-en package is 3.4 MB untarred). I've always wondered if conda should just package the dictionaries with the package. But I also only personally care about -en so that's the only one I've ever packaged.